### ISSUE TITLE
Swift C++ interoperability

### ISSUE THREAD

**Issue by arguiot** [Boilertalk/secp256k1.swift#20]
`secp256k1` is not working when C++/ObjectiveC++ interoperability is enabled with Swift 5.9.
<img width="932" alt="Capture d’écran 2023-07-20 à 10 09 34" src="https://github.com/Boilertalk/secp256k1.swift/assets/18022260/ff8a5c46-6495-44e0-9096-fa005492c5c9">
I get: `Cannot find 'SECP256K1_CONTEXT_SIGN' in scope`

**Comment by koraykoska** [comment]
No idea why. I didn’t know Swift 5.9 existed. Is this a pre-release?
Do you know why this is happening?

**Comment by arguiot** [comment]
Yes, it's a pre-release actually. Things works well when C++ interoperability is disabled. But when enabled, things starts to get weird...
